app/instance-initializers/ember-href-to.js

```diff
@@ -0,0 +1,39 @@
+import Em from 'ember';
+
+function _getNormalisedRootUrl(router) {
+  var rootURL = router.rootURL;
+  if(rootURL.charAt(rootURL.length - 1) !== '/') {
+    rootURL = rootURL + '/';
+  }
+  return rootURL;
+}
+
+let router;
+
+export default {
+  name: 'ember-href-to',
+  initialize: function(applicationInstance) {
+    router = applicationInstance.container.lookup('router:main');
```

Comment:
Oh, we're leaking this event handler between test runs. Something like this might be a little better:

```javascript
  initialize: function(applicationInstance) {
    let router = applicationInstance.container.lookup('router:main');
    let rootURL = _getNormalisedRootUrl(router);

    Em.$(document.body).off('click.href-to', 'a');
    Em.$(document.body).on('click.href-to', 'a', function(e) {
      let $target = Em.$(e.currentTarget);
      let handleClick = (e.which === 1 && !e.ctrlKey && !e.metaKey);

      if(handleClick && !$target.hasClass('ember-view') && Em.isNone($target.attr('data-ember-action'))) {
        let url = $target.attr('href');

        if(url && url.indexOf(rootURL) === 0) {
          url = url.substr(rootURL.length - 1);

          if(router.router.recognizer.recognize(url)) {
            router.handleURL(url);
            router.router.updateURL(url);
            return false;
          }
        }
      }
      return true;
    });
  }
```